### PR TITLE
Darker and sexier theme (fix #245)

### LINKED ISSUE #245: Dark theme could be darker
Just tried the dark theme in the dark, and I found it a little too light.

Maybe make it darker, with some blue/night nuance

### PR DESCRIPTION
**Pull request by KeitIG**

### CODE REVIEW

**Comment by YurySolovyov** [comment]
Any screenshots?

File: src/styles/themes/theme-dark.scss [hunk 1]
```diff
@@ -85,26 +85,26 @@ body.theme-dark {
...
-                       background-color: #555;
+                       background-color: #33393b;
 
                        .queue-infos {
                            color: #DDD;
                        }
 
                        .empty-button {
-                           background-color: #555;
+                           background-color: #33393b;
```
Comment (YurySolovyov) [inline src/styles/themes/theme-dark.scss]: these seem to be quite repetitive, can we move them to variables?

**Comment by KeitIG** [comment]
## old

museeksold

## new

museeksnew

About the variables things, "ye I should" but I feel very lazy right now and will do it later.

**Comment by YurySolovyov** [comment]
Also, no changes the light theme?

**Comment by KeitIG** [comment]
Nope, do you see something wrong in particular ?

**Pull request merged**